File: briareos-master/dev/project/briareos/zcluster.py
from briareos.core import worker_manager

from briareos.utils import logger
from briareos.utils import loader
from briareos.common import *


class ZCluster:
    def __init__(self, cluster_config_path=ZCLUSTER_CONFIG_PATH):
        logger.info("Initializing Briareos Z-Cluster")
        self.config = loader.import_zcluster_config(cluster_config_path)
        self.cluster = worker_manager.Cluster(self.config)

    def start(self):
        self.cluster.start()
        logger.info("Z-Cluster ID: %s" % self.cluster.id)
        logger.success("Briareos Z-Cluster is running")

    def stop(self):
        logger.info("Stopping Briareos Z-Cluster")
        self.cluster.stop()

File: jbriareos-master/Tadeu_Freitas-jbriareos-c16b99c2c2d9/src/main/java/components/ZCluster.java
package components;

import core.Cluster;

public class ZCluster {
    public Cluster cluster;

    public ZCluster() {
        System.out.println("Initializing Briareos ZCluster");
        this.cluster = new Cluster();
    }

    public boolean start() {
        if(!this.cluster.start())
            return false;

        System.out.println("ZCluster ID: " + this.cluster.clusterId);
        return true;
    }

    public void stop() {
        System.out.println("Stopping Briareos ZCluster " + this.cluster.clusterId);
        this.cluster.stop();
    }
}
